File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.py
class Solution:
    def firstPalindrome(self, words: List[str]) -> str:
        arr = ""
        for element in words:
            n = len(element) // 2
            count = 0
            for character in range(n):
                if element[character] == element[-character-1]:
                    count += 1
            if count == n:
                return element
        return arr 

File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.java
class Solution {
    public String firstPalindrome(String[] words) {
        int count = 0;
        for (String elem:words){
            int left = 0;
            int right = elem.length()-1;
            int control = 0;
            while (left < right){
                if (elem.charAt(left) != elem.charAt(right)){
                    control = 1;
                    break;
                }
                left += 1;
                right -= 1;
            }
            if (control == 0){
                return elem;
            }
        }
        return ""; 
    }
}